### PR TITLE
Task/sql schema

### PR DESCRIPTION
**Pull request by rorysproule1**
## Description
Changes made for this branch

## Checklist
- [x] All manual tests passing
- [ ] All unit tests passing 
- [ ] All UI tests passing
- [ ] Code reviewed by peer:\
&nbsp; - *No redundant whitespace*\
&nbsp; - *Indentation correct*\
&nbsp; - *Naming conventions followed*\
&nbsp; - *Comments added (where needed)*\
&nbsp; - *No commented out redundant code*\
&nbsp; - *Branch name correct*
- [ ] Code reviewed by TA

### CODE REVIEW

**Comment by jurassic-marc** [comment]
Seems fine

File: schema.sql [hunk 1]
```diff
@@ -116,6 +117,34 @@ band.band_id, band.name AS 'BandName'
...
+
+    -- increment the priority of the bands being moved 
+    UPDATE band SET band.priority_in_school = band.priority_in_school + 1
+    WHERE band.school_id = new_school_id AND band.priority_in_school >= new_priority_in_school;
+
+    -- add new band
+    INSERT INTO band(name, school_id, priority_in_school, description_id) 
+    VALUES (new_name, new_school_id, new_priority_in_school, desc_id);
+END;//
+DELIMITER ;
```
Comment (matthewdoreilly) [inline schema.sql]: Noice 👍

File: schema.sql [hunk 2]
```diff
@@ -197,7 +226,7 @@ VALUES ('Lead Software Engineer', 2, 5, 'Expert in their field, is consulted by
...
 ('Apprentice Software Engineer', 2, 9, 'Here primarily to learn; contributes to projects.', null);
 INSERT INTO role (name, department_id, band_id, summary, job_spec_url)
-VALUES ('Senior Security Architect', 2, 4, 'Serves the company\'s commercial and delivery interests, owns single initiatives or projects, advocates effective coaching and ensures that it happens.', 'https://kainossoftwareltd.sharepoint.com/:b:/r/people/Shared%20Documents/Job%20Descriptions/Cyber%20Security/Job%20Specification%20-%20Senior%20Security%20Architect%20-%20Manager.pdf?csf=1&e=DJdlxg'), 
+VALUES ('Senior Security Architect', 2, 4, 'Serves the companys commercial and delivery interests, owns single initiatives or projects, advocates effective coaching and ensures that it happens.', 'https://kainossoftwareltd.sharepoint.com/:b:/r/people/Shared%20Documents/Job%20Descriptions/Cyber%20Security/Job%20Specification%20-%20Senior%20Security%20Architect%20-%20Manager.pdf?csf=1&e=DJdlxg'),
```
Comment (matthewdoreilly) [inline schema.sql]: Why this change? Apostrophe should be here
